{
  "stats": {
    "submitted": 100,
    "finished": 100,
    "running": 0
  },
  "data": [
    {
      "config_id": 1,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.615483,
      "status": 2,
      "starttime": 1747735474.132656,
      "endtime": 1747735474.134302,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 2,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.617618,
      "status": 2,
      "starttime": 1747735474.13538,
      "endtime": 1747735474.136867,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 3,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.620885,
      "status": 2,
      "starttime": 1747735474.155752,
      "endtime": 1747735474.156725,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 1,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.624127,
      "status": 2,
      "starttime": 1747735474.177888,
      "endtime": 1747735474.178865,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 4,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.62556,
      "status": 2,
      "starttime": 1747735474.197239,
      "endtime": 1747735474.198303,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 1,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.630199,
      "status": 2,
      "starttime": 1747735474.220159,
      "endtime": 1747735474.221293,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 4,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.633672,
      "status": 2,
      "starttime": 1747735474.2349062,
      "endtime": 1747735474.2359538,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 5,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.637818,
      "status": 2,
      "starttime": 1747735474.255577,
      "endtime": 1747735474.257165,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 4,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.64322,
      "status": 2,
      "starttime": 1747735474.276635,
      "endtime": 1747735474.278355,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 5,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.721825,
      "status": 2,
      "starttime": 1747735474.371559,
      "endtime": 1747735474.3732858,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 6,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.726823,
      "status": 2,
      "starttime": 1747735474.394054,
      "endtime": 1747735474.395155,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 5,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.730843,
      "status": 2,
      "starttime": 1747735474.414859,
      "endtime": 1747735474.4158652,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 6,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.734869,
      "status": 2,
      "starttime": 1747735474.435196,
      "endtime": 1747735474.436219,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 7,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.739074,
      "status": 2,
      "starttime": 1747735474.457254,
      "endtime": 1747735474.458369,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 6,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.74396,
      "status": 2,
      "starttime": 1747735474.4798489,
      "endtime": 1747735474.4809852,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 7,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.748301,
      "status": 2,
      "starttime": 1747735474.5007858,
      "endtime": 1747735474.502437,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 8,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.752449,
      "status": 2,
      "starttime": 1747735474.522018,
      "endtime": 1747735474.523148,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 7,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.757019,
      "status": 2,
      "starttime": 1747735474.543572,
      "endtime": 1747735474.544731,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 8,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.7614,
      "status": 2,
      "starttime": 1747735474.564815,
      "endtime": 1747735474.566436,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 9,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.765412,
      "status": 2,
      "starttime": 1747735474.58687,
      "endtime": 1747735474.5885189,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 8,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.769522,
      "status": 2,
      "starttime": 1747735474.6077929,
      "endtime": 1747735474.609539,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 9,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.774337,
      "status": 2,
      "starttime": 1747735474.6291149,
      "endtime": 1747735474.630143,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 10,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.778516,
      "status": 2,
      "starttime": 1747735474.651086,
      "endtime": 1747735474.6526709,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 9,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.78258,
      "status": 2,
      "starttime": 1747735474.672093,
      "endtime": 1747735474.673731,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 10,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.786945,
      "status": 2,
      "starttime": 1747735474.692619,
      "endtime": 1747735474.693804,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 11,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.791285,
      "status": 2,
      "starttime": 1747735474.715208,
      "endtime": 1747735474.7168021,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 10,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.795386,
      "status": 2,
      "starttime": 1747735474.7368262,
      "endtime": 1747735474.738409,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 11,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.799649,
      "status": 2,
      "starttime": 1747735474.756512,
      "endtime": 1747735474.7580101,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 12,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.803437,
      "status": 2,
      "starttime": 1747735474.778338,
      "endtime": 1747735474.7799082,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 11,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.807752,
      "status": 2,
      "starttime": 1747735474.7986848,
      "endtime": 1747735474.799676,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 12,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.811733,
      "status": 2,
      "starttime": 1747735474.821053,
      "endtime": 1747735474.822057,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 13,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.814285,
      "status": 2,
      "starttime": 1747735474.833706,
      "endtime": 1747735474.834739,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 12,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.818646,
      "status": 2,
      "starttime": 1747735474.854523,
      "endtime": 1747735474.8554392,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 13,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.822542,
      "status": 2,
      "starttime": 1747735474.877291,
      "endtime": 1747735474.878309,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 14,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.825839,
      "status": 2,
      "starttime": 1747735474.890445,
      "endtime": 1747735474.891949,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 13,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.829629,
      "status": 2,
      "starttime": 1747735474.911009,
      "endtime": 1747735474.912008,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 14,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.834199,
      "status": 2,
      "starttime": 1747735474.931347,
      "endtime": 1747735474.9324212,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 15,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.838145,
      "status": 2,
      "starttime": 1747735474.951746,
      "endtime": 1747735474.9527712,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 14,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.843396,
      "status": 2,
      "starttime": 1747735474.9749582,
      "endtime": 1747735474.975961,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 15,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.846104,
      "status": 2,
      "starttime": 1747735474.9883351,
      "endtime": 1747735474.9893272,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 16,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.849759,
      "status": 2,
      "starttime": 1747735475.00997,
      "endtime": 1747735475.011549,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 15,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.853516,
      "status": 2,
      "starttime": 1747735475.030039,
      "endtime": 1747735475.03105,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 16,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.857218,
      "status": 2,
      "starttime": 1747735475.050271,
      "endtime": 1747735475.0512571,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 17,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.860993,
      "status": 2,
      "starttime": 1747735475.072298,
      "endtime": 1747735475.073768,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 16,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.86531,
      "status": 2,
      "starttime": 1747735475.092698,
      "endtime": 1747735475.0937102,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 17,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.868997,
      "status": 2,
      "starttime": 1747735475.112691,
      "endtime": 1747735475.113618,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 18,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.872603,
      "status": 2,
      "starttime": 1747735475.134291,
      "endtime": 1747735475.1357608,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 17,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.876287,
      "status": 2,
      "starttime": 1747735475.154083,
      "endtime": 1747735475.155045,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 18,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.880765,
      "status": 2,
      "starttime": 1747735475.174599,
      "endtime": 1747735475.175504,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 19,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.884526,
      "status": 2,
      "starttime": 1747735475.195167,
      "endtime": 1747735475.196071,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 18,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.888584,
      "status": 2,
      "starttime": 1747735475.217925,
      "endtime": 1747735475.218931,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 19,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.891478,
      "status": 2,
      "starttime": 1747735475.2309701,
      "endtime": 1747735475.232022,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 20,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.895195,
      "status": 2,
      "starttime": 1747735475.2532501,
      "endtime": 1747735475.254823,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 19,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.89899,
      "status": 2,
      "starttime": 1747735475.273301,
      "endtime": 1747735475.2753909,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 20,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.903248,
      "status": 2,
      "starttime": 1747735475.294105,
      "endtime": 1747735475.29508,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 21,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.907558,
      "status": 2,
      "starttime": 1747735475.3178759,
      "endtime": 1747735475.318977,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 20,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.913269,
      "status": 2,
      "starttime": 1747735475.37511,
      "endtime": 1747735475.376195,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 21,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.917534,
      "status": 2,
      "starttime": 1747735475.3973,
      "endtime": 1747735475.398258,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 22,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.921692,
      "status": 2,
      "starttime": 1747735475.4213052,
      "endtime": 1747735475.4223099,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 21,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.925969,
      "status": 2,
      "starttime": 1747735475.443903,
      "endtime": 1747735475.4448798,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 22,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.930776,
      "status": 2,
      "starttime": 1747735475.46647,
      "endtime": 1747735475.468162,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 23,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.934669,
      "status": 2,
      "starttime": 1747735475.4903061,
      "endtime": 1747735475.491307,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 22,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.938567,
      "status": 2,
      "starttime": 1747735475.511437,
      "endtime": 1747735475.512416,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 23,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.942312,
      "status": 2,
      "starttime": 1747735475.5320802,
      "endtime": 1747735475.533021,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 24,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.954006,
      "status": 2,
      "starttime": 1747735476.357606,
      "endtime": 1747735476.358597,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 23,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.958595,
      "status": 2,
      "starttime": 1747735476.379724,
      "endtime": 1747735476.380708,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 24,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.962355,
      "status": 2,
      "starttime": 1747735476.401505,
      "endtime": 1747735476.403009,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 25,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.966059,
      "status": 2,
      "starttime": 1747735476.4241989,
      "endtime": 1747735476.425177,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 24,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.969861,
      "status": 2,
      "starttime": 1747735476.445437,
      "endtime": 1747735476.4469178,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 25,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.974188,
      "status": 2,
      "starttime": 1747735476.467595,
      "endtime": 1747735476.468554,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 26,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.978009,
      "status": 2,
      "starttime": 1747735476.491562,
      "endtime": 1747735476.4925141,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 25,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.98162,
      "status": 2,
      "starttime": 1747735476.51221,
      "endtime": 1747735476.513154,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 26,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.985981,
      "status": 2,
      "starttime": 1747735476.533798,
      "endtime": 1747735476.535429,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 27,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.989111,
      "status": 2,
      "starttime": 1747735476.556266,
      "endtime": 1747735476.557201,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 26,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.992934,
      "status": 2,
      "starttime": 1747735476.577565,
      "endtime": 1747735476.57906,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 27,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 3.996674,
      "status": 2,
      "starttime": 1747735476.598956,
      "endtime": 1747735476.599936,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 28,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.000456,
      "status": 2,
      "starttime": 1747735476.622221,
      "endtime": 1747735476.623177,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 27,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.004858,
      "status": 2,
      "starttime": 1747735476.644201,
      "endtime": 1747735476.645139,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 28,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.008686,
      "status": 2,
      "starttime": 1747735476.6661289,
      "endtime": 1747735476.667634,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 29,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.012456,
      "status": 2,
      "starttime": 1747735476.689752,
      "endtime": 1747735476.6907132,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 28,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.017045,
      "status": 2,
      "starttime": 1747735476.7112021,
      "endtime": 1747735476.7129102,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 29,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.020186,
      "status": 2,
      "starttime": 1747735476.7318962,
      "endtime": 1747735476.73339,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 30,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.023832,
      "status": 2,
      "starttime": 1747735476.754955,
      "endtime": 1747735476.756456,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 29,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.027516,
      "status": 2,
      "starttime": 1747735476.776125,
      "endtime": 1747735476.777556,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 30,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.031372,
      "status": 2,
      "starttime": 1747735476.798858,
      "endtime": 1747735476.800385,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 31,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.035695,
      "status": 2,
      "starttime": 1747735476.825002,
      "endtime": 1747735476.8261251,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 30,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.039893,
      "status": 2,
      "starttime": 1747735476.848526,
      "endtime": 1747735476.849499,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 31,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.044058,
      "status": 2,
      "starttime": 1747735476.872322,
      "endtime": 1747735476.873358,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 32,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.058939,
      "status": 2,
      "starttime": 1747735477.776337,
      "endtime": 1747735477.7774432,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 31,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.064085,
      "status": 2,
      "starttime": 1747735477.801823,
      "endtime": 1747735477.802795,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 32,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.069529,
      "status": 2,
      "starttime": 1747735477.9132092,
      "endtime": 1747735477.914221,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 33,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.073661,
      "status": 2,
      "starttime": 1747735477.938648,
      "endtime": 1747735477.9402041,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 32,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.078907,
      "status": 2,
      "starttime": 1747735477.963203,
      "endtime": 1747735477.96426,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 33,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.082846,
      "status": 2,
      "starttime": 1747735477.9886148,
      "endtime": 1747735477.990135,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 34,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.086938,
      "status": 2,
      "starttime": 1747735478.0141098,
      "endtime": 1747735478.01506,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 33,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.090726,
      "status": 2,
      "starttime": 1747735478.036045,
      "endtime": 1747735478.037003,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 34,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.094579,
      "status": 2,
      "starttime": 1747735478.058094,
      "endtime": 1747735478.0596328,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 35,
      "instance": null,
      "seed": 209652396,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.098376,
      "status": 2,
      "starttime": 1747735478.0829592,
      "endtime": 1747735478.083922,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 34,
      "instance": null,
      "seed": 924231285,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.102089,
      "status": 2,
      "starttime": 1747735478.105105,
      "endtime": 1747735478.106604,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    },
    {
      "config_id": 35,
      "instance": null,
      "seed": 398764591,
      "budget": null,
      "cost": Infinity,
      "time": 0.0,
      "cpu_time": 4.106312,
      "status": 2,
      "starttime": 1747735478.127649,
      "endtime": 1747735478.129406,
      "additional_info": {
        "traceback": "Traceback (most recent call last):\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 190, in run\n    rval = self(config_copy, target_function, kwargs)\n  File \"/opt/anaconda3/envs/SMAC/lib/python3.10/site-packages/smac/runner/target_function_runner.py\", line 264, in __call__\n    return algorithm(config, **algorithm_kwargs)\nTypeError: AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\n",
        "error": "TypeError(\"AutoMLAgent.run_scenario.<locals>.smac_train_function() got multiple values for argument 'seed'\")"
      }
    }
  ],
  "configs": {
    "1": {
      "classifier": "knn",
      "knn_algorithm": "kd_tree",
      "leaf_size": 6,
      "n_neighbors": 5,
      "p": 2,
      "rf_max_depth": 50,
      "rf_max_features": null,
      "rf_min_samples_leaf": 20,
      "rf_min_samples_split": 16,
      "rf_n_estimators": 146,
      "svm_C": 0.0056074086723,
      "svm_coef0": 0.6567580401897,
      "svm_degree": 3,
      "svm_gamma": 3.86698833e-05,
      "svm_kernel": "sigmoid",
      "weights": "distance"
    },
    "2": {
      "classifier": "random_forest",
      "knn_algorithm": "auto",
      "leaf_size": 48,
      "n_neighbors": 18,
      "p": 1,
      "rf_max_depth": 72,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 5,
      "rf_min_samples_split": 3,
      "rf_n_estimators": 76,
      "svm_C": 0.2635499122062,
      "svm_coef0": -0.8251485805959,
      "svm_degree": 4,
      "svm_gamma": 0.0165271107674,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "3": {
      "classifier": "random_forest",
      "knn_algorithm": "brute",
      "leaf_size": 22,
      "n_neighbors": 6,
      "p": 1,
      "rf_max_depth": 97,
      "rf_max_features": null,
      "rf_min_samples_leaf": 10,
      "rf_min_samples_split": 11,
      "rf_n_estimators": 37,
      "svm_C": 0.6151414328209,
      "svm_coef0": -0.0472089871764,
      "svm_degree": 3,
      "svm_gamma": 0.0014673891059,
      "svm_kernel": "linear",
      "weights": "distance"
    },
    "4": {
      "classifier": "knn",
      "knn_algorithm": "ball_tree",
      "leaf_size": 26,
      "n_neighbors": 13,
      "p": 2,
      "rf_max_depth": 20,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 15,
      "rf_min_samples_split": 15,
      "rf_n_estimators": 160,
      "svm_C": 1.34902715e-05,
      "svm_coef0": 0.4649700615555,
      "svm_degree": 4,
      "svm_gamma": 0.6105170555979,
      "svm_kernel": "poly",
      "weights": "uniform"
    },
    "5": {
      "classifier": "knn",
      "knn_algorithm": "brute",
      "leaf_size": 42,
      "n_neighbors": 13,
      "p": 2,
      "rf_max_depth": 32,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 6,
      "rf_min_samples_split": 8,
      "rf_n_estimators": 90,
      "svm_C": 3.3465917160193,
      "svm_coef0": 0.1408106088638,
      "svm_degree": 3,
      "svm_gamma": 0.0729307892201,
      "svm_kernel": "poly",
      "weights": "uniform"
    },
    "6": {
      "classifier": "svm",
      "knn_algorithm": "ball_tree",
      "leaf_size": 12,
      "n_neighbors": 10,
      "p": 1,
      "rf_max_depth": 59,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 12,
      "rf_min_samples_split": 13,
      "rf_n_estimators": 107,
      "svm_C": 6.41286009e-05,
      "svm_coef0": -0.3713616412133,
      "svm_degree": 5,
      "svm_gamma": 0.0001706426365,
      "svm_kernel": "linear",
      "weights": "distance"
    },
    "7": {
      "classifier": "random_forest",
      "knn_algorithm": "kd_tree",
      "leaf_size": 32,
      "n_neighbors": 17,
      "p": 1,
      "rf_max_depth": 85,
      "rf_max_features": null,
      "rf_min_samples_leaf": 16,
      "rf_min_samples_split": 19,
      "rf_n_estimators": 199,
      "svm_C": 0.0008648993186,
      "svm_coef0": -0.5008622929454,
      "svm_degree": 2,
      "svm_gamma": 1.9472512715362,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "8": {
      "classifier": "svm",
      "knn_algorithm": "auto",
      "leaf_size": 16,
      "n_neighbors": 2,
      "p": 2,
      "rf_max_depth": 13,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 2,
      "rf_min_samples_split": 5,
      "rf_n_estimators": 22,
      "svm_C": 0.0440777809813,
      "svm_coef0": 0.9810356292874,
      "svm_degree": 4,
      "svm_gamma": 0.0046802315488,
      "svm_kernel": "sigmoid",
      "weights": "distance"
    },
    "9": {
      "classifier": "svm",
      "knn_algorithm": "brute",
      "leaf_size": 36,
      "n_neighbors": 8,
      "p": 1,
      "rf_max_depth": 16,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 7,
      "rf_min_samples_split": 7,
      "rf_n_estimators": 171,
      "svm_C": 0.0115232416041,
      "svm_coef0": 0.5674329828471,
      "svm_degree": 5,
      "svm_gamma": 7.0803444344705,
      "svm_kernel": "linear",
      "weights": "uniform"
    },
    "10": {
      "classifier": "random_forest",
      "knn_algorithm": "ball_tree",
      "leaf_size": 18,
      "n_neighbors": 15,
      "p": 2,
      "rf_max_depth": 88,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 13,
      "rf_min_samples_split": 12,
      "rf_n_estimators": 50,
      "svm_C": 0.0005873120205,
      "svm_coef0": -0.9155110009015,
      "svm_degree": 3,
      "svm_gamma": 0.0030267241459,
      "svm_kernel": "poly",
      "weights": "distance"
    },
    "11": {
      "classifier": "svm",
      "knn_algorithm": "kd_tree",
      "leaf_size": 38,
      "n_neighbors": 3,
      "p": 2,
      "rf_max_depth": 69,
      "rf_max_features": null,
      "rf_min_samples_leaf": 18,
      "rf_min_samples_split": 20,
      "rf_n_estimators": 66,
      "svm_C": 0.0002507398077,
      "svm_coef0": -0.2088604811579,
      "svm_degree": 4,
      "svm_gamma": 0.2674861092024,
      "svm_kernel": "sigmoid",
      "weights": "uniform"
    },
    "12": {
      "classifier": "random_forest",
      "knn_algorithm": "auto",
      "leaf_size": 29,
      "n_neighbors": 4,
      "p": 2,
      "rf_max_depth": 81,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 4,
      "rf_min_samples_split": 4,
      "rf_n_estimators": 185,
      "svm_C": 1.0108374005033,
      "svm_coef0": -0.4002006240189,
      "svm_degree": 3,
      "svm_gamma": 0.9313959710406,
      "svm_kernel": "sigmoid",
      "weights": "uniform"
    },
    "13": {
      "classifier": "random_forest",
      "knn_algorithm": "brute",
      "leaf_size": 4,
      "n_neighbors": 11,
      "p": 2,
      "rf_max_depth": 51,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 8,
      "rf_min_samples_split": 10,
      "rf_n_estimators": 117,
      "svm_C": 0.0747087244885,
      "svm_coef0": -0.7240372244269,
      "svm_degree": 5,
      "svm_gamma": 1.06866933e-05,
      "svm_kernel": "poly",
      "weights": "distance"
    },
    "14": {
      "classifier": "svm",
      "knn_algorithm": "ball_tree",
      "leaf_size": 44,
      "n_neighbors": 8,
      "p": 1,
      "rf_max_depth": 28,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 14,
      "rf_min_samples_split": 14,
      "rf_n_estimators": 104,
      "svm_C": 0.0035116566217,
      "svm_coef0": 0.7588980533183,
      "svm_degree": 3,
      "svm_gamma": 0.0256911232882,
      "svm_kernel": "linear",
      "weights": "uniform"
    },
    "15": {
      "classifier": "knn",
      "knn_algorithm": "brute",
      "leaf_size": 16,
      "n_neighbors": 17,
      "p": 2,
      "rf_max_depth": 78,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 9,
      "rf_min_samples_split": 11,
      "rf_n_estimators": 116,
      "svm_C": 0.0021670116638,
      "svm_coef0": 0.2345704454929,
      "svm_degree": 4,
      "svm_gamma": 0.124631321997,
      "svm_kernel": "poly",
      "weights": "distance"
    },
    "16": {
      "classifier": "random_forest",
      "knn_algorithm": "ball_tree",
      "leaf_size": 37,
      "n_neighbors": 1,
      "p": 1,
      "rf_max_depth": 1,
      "rf_max_features": null,
      "rf_min_samples_leaf": 13,
      "rf_min_samples_split": 8,
      "rf_n_estimators": 82,
      "svm_C": 0.0988042175812,
      "svm_coef0": -0.2776095569134,
      "svm_degree": 2,
      "svm_gamma": 0.0002996016838,
      "svm_kernel": "linear",
      "weights": "uniform"
    },
    "17": {
      "classifier": "svm",
      "knn_algorithm": "kd_tree",
      "leaf_size": 8,
      "n_neighbors": 14,
      "p": 1,
      "rf_max_depth": 31,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 18,
      "rf_min_samples_split": 6,
      "rf_n_estimators": 32,
      "svm_C": 1.3379510341854,
      "svm_coef0": -0.5946106817573,
      "svm_degree": 5,
      "svm_gamma": 3.3533467357224,
      "svm_kernel": "rbf",
      "weights": "distance"
    },
    "18": {
      "classifier": "knn",
      "knn_algorithm": "auto",
      "leaf_size": 40,
      "n_neighbors": 10,
      "p": 2,
      "rf_max_depth": 54,
      "rf_max_features": null,
      "rf_min_samples_leaf": 4,
      "rf_min_samples_split": 19,
      "rf_n_estimators": 191,
      "svm_C": 2.84642338e-05,
      "svm_coef0": 0.8872949853539,
      "svm_degree": 3,
      "svm_gamma": 0.007844007096,
      "svm_kernel": "sigmoid",
      "weights": "uniform"
    },
    "19": {
      "classifier": "knn",
      "knn_algorithm": "kd_tree",
      "leaf_size": 31,
      "n_neighbors": 7,
      "p": 2,
      "rf_max_depth": 91,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 16,
      "rf_min_samples_split": 4,
      "rf_n_estimators": 74,
      "svm_C": 8.667036348696,
      "svm_coef0": 0.6880139429122,
      "svm_degree": 4,
      "svm_gamma": 2.84621346e-05,
      "svm_kernel": "sigmoid",
      "weights": "uniform"
    },
    "20": {
      "classifier": "svm",
      "knn_algorithm": "auto",
      "leaf_size": 23,
      "n_neighbors": 12,
      "p": 1,
      "rf_max_depth": 19,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 3,
      "rf_min_samples_split": 18,
      "rf_n_estimators": 149,
      "svm_C": 0.0001700499059,
      "svm_coef0": -0.7938849329948,
      "svm_degree": 3,
      "svm_gamma": 0.011839906546,
      "svm_kernel": "rbf",
      "weights": "distance"
    },
    "21": {
      "classifier": "random_forest",
      "knn_algorithm": "brute",
      "leaf_size": 46,
      "n_neighbors": 5,
      "p": 1,
      "rf_max_depth": 38,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 8,
      "rf_min_samples_split": 14,
      "rf_n_estimators": 158,
      "svm_C": 0.000398657247,
      "svm_coef0": -0.0784458126873,
      "svm_degree": 4,
      "svm_gamma": 0.0010432248653,
      "svm_kernel": "linear",
      "weights": "uniform"
    },
    "22": {
      "classifier": "svm",
      "knn_algorithm": "ball_tree",
      "leaf_size": 2,
      "n_neighbors": 18,
      "p": 2,
      "rf_max_depth": 65,
      "rf_max_features": null,
      "rf_min_samples_leaf": 12,
      "rf_min_samples_split": 9,
      "rf_n_estimators": 41,
      "svm_C": 0.0208022626696,
      "svm_coef0": 0.4337254837155,
      "svm_degree": 3,
      "svm_gamma": 0.4459838466846,
      "svm_kernel": "poly",
      "weights": "distance"
    },
    "23": {
      "classifier": "svm",
      "knn_algorithm": "kd_tree",
      "leaf_size": 50,
      "n_neighbors": 12,
      "p": 1,
      "rf_max_depth": 62,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 16,
      "rf_min_samples_split": 3,
      "rf_n_estimators": 178,
      "svm_C": 0.0299185459378,
      "svm_coef0": 0.0212219469249,
      "svm_degree": 3,
      "svm_gamma": 0.0006792597028,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "24": {
      "classifier": "random_forest",
      "knn_algorithm": "kd_tree",
      "leaf_size": 50,
      "n_neighbors": 9,
      "p": 1,
      "rf_max_depth": 55,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 9,
      "rf_min_samples_split": 4,
      "rf_n_estimators": 178,
      "svm_C": 0.0742261818638,
      "svm_coef0": 0.0212219469249,
      "svm_degree": 5,
      "svm_gamma": 0.0006792597028,
      "svm_kernel": "sigmoid",
      "weights": "uniform"
    },
    "25": {
      "classifier": "svm",
      "knn_algorithm": "kd_tree",
      "leaf_size": 5,
      "n_neighbors": 12,
      "p": 1,
      "rf_max_depth": 19,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 6,
      "rf_min_samples_split": 6,
      "rf_n_estimators": 32,
      "svm_C": 1.3379510341854,
      "svm_coef0": -0.5946106817573,
      "svm_degree": 5,
      "svm_gamma": 2.6430200451881,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "26": {
      "classifier": "knn",
      "knn_algorithm": "ball_tree",
      "leaf_size": 39,
      "n_neighbors": 1,
      "p": 2,
      "rf_max_depth": 86,
      "rf_max_features": null,
      "rf_min_samples_leaf": 14,
      "rf_min_samples_split": 12,
      "rf_n_estimators": 116,
      "svm_C": 0.2160233943305,
      "svm_coef0": -0.4584821904886,
      "svm_degree": 4,
      "svm_gamma": 1.23168873e-05,
      "svm_kernel": "rbf",
      "weights": "distance"
    },
    "27": {
      "classifier": "svm",
      "knn_algorithm": "kd_tree",
      "leaf_size": 5,
      "n_neighbors": 15,
      "p": 1,
      "rf_max_depth": 56,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 3,
      "rf_min_samples_split": 9,
      "rf_n_estimators": 112,
      "svm_C": 3.2941232521713,
      "svm_coef0": 0.7260407258179,
      "svm_degree": 3,
      "svm_gamma": 6.0763467986434,
      "svm_kernel": "poly",
      "weights": "distance"
    },
    "28": {
      "classifier": "knn",
      "knn_algorithm": "auto",
      "leaf_size": 42,
      "n_neighbors": 3,
      "p": 2,
      "rf_max_depth": 1,
      "rf_max_features": null,
      "rf_min_samples_leaf": 9,
      "rf_min_samples_split": 8,
      "rf_n_estimators": 90,
      "svm_C": 3.3465917160193,
      "svm_coef0": 0.1408106088638,
      "svm_degree": 3,
      "svm_gamma": 0.0729307892201,
      "svm_kernel": "poly",
      "weights": "uniform"
    },
    "29": {
      "classifier": "random_forest",
      "knn_algorithm": "ball_tree",
      "leaf_size": 16,
      "n_neighbors": 17,
      "p": 1,
      "rf_max_depth": 77,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 6,
      "rf_min_samples_split": 11,
      "rf_n_estimators": 116,
      "svm_C": 0.0021670116638,
      "svm_coef0": 0.2189730898461,
      "svm_degree": 2,
      "svm_gamma": 0.124631321997,
      "svm_kernel": "poly",
      "weights": "distance"
    },
    "30": {
      "classifier": "knn",
      "knn_algorithm": "ball_tree",
      "leaf_size": 25,
      "n_neighbors": 13,
      "p": 2,
      "rf_max_depth": 20,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 15,
      "rf_min_samples_split": 11,
      "rf_n_estimators": 160,
      "svm_C": 1.34902715e-05,
      "svm_coef0": 0.5363975297481,
      "svm_degree": 4,
      "svm_gamma": 1.1093727240935,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "31": {
      "classifier": "random_forest",
      "knn_algorithm": "kd_tree",
      "leaf_size": 27,
      "n_neighbors": 16,
      "p": 1,
      "rf_max_depth": 80,
      "rf_max_features": "sqrt",
      "rf_min_samples_leaf": 16,
      "rf_min_samples_split": 19,
      "rf_n_estimators": 174,
      "svm_C": 0.0008648993186,
      "svm_coef0": -0.4193090496876,
      "svm_degree": 3,
      "svm_gamma": 1.9472512715362,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "32": {
      "classifier": "svm",
      "knn_algorithm": "kd_tree",
      "leaf_size": 6,
      "n_neighbors": 5,
      "p": 2,
      "rf_max_depth": 51,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 20,
      "rf_min_samples_split": 15,
      "rf_n_estimators": 146,
      "svm_C": 0.0030322155217,
      "svm_coef0": 0.6567580401897,
      "svm_degree": 2,
      "svm_gamma": 3.86698833e-05,
      "svm_kernel": "sigmoid",
      "weights": "distance"
    },
    "33": {
      "classifier": "random_forest",
      "knn_algorithm": "kd_tree",
      "leaf_size": 42,
      "n_neighbors": 17,
      "p": 2,
      "rf_max_depth": 85,
      "rf_max_features": "log2",
      "rf_min_samples_leaf": 2,
      "rf_min_samples_split": 7,
      "rf_n_estimators": 135,
      "svm_C": 0.0011797295969,
      "svm_coef0": 0.1256459301923,
      "svm_degree": 5,
      "svm_gamma": 0.1219603854514,
      "svm_kernel": "sigmoid",
      "weights": "uniform"
    },
    "34": {
      "classifier": "random_forest",
      "knn_algorithm": "ball_tree",
      "leaf_size": 24,
      "n_neighbors": 6,
      "p": 1,
      "rf_max_depth": 75,
      "rf_max_features": null,
      "rf_min_samples_leaf": 10,
      "rf_min_samples_split": 11,
      "rf_n_estimators": 25,
      "svm_C": 0.6151414328209,
      "svm_coef0": 0.1500306988036,
      "svm_degree": 5,
      "svm_gamma": 0.0014673891059,
      "svm_kernel": "rbf",
      "weights": "uniform"
    },
    "35": {
      "classifier": "svm",
      "knn_algorithm": "ball_tree",
      "leaf_size": 28,
      "n_neighbors": 19,
      "p": 1,
      "rf_max_depth": 73,
      "rf_max_features": null,
      "rf_min_samples_leaf": 5,
      "rf_min_samples_split": 3,
      "rf_n_estimators": 184,
      "svm_C": 0.8322352655612,
      "svm_coef0": -0.5160680369506,
      "svm_degree": 5,
      "svm_gamma": 0.0372082353434,
      "svm_kernel": "rbf",
      "weights": "uniform"
    }
  },
  "config_origins": {
    "1": "Initial Design: Sobol",
    "2": "Initial Design: Sobol",
    "3": "Initial Design: Sobol",
    "4": "Initial Design: Sobol",
    "5": "Initial Design: Sobol",
    "6": "Initial Design: Sobol",
    "7": "Initial Design: Sobol",
    "8": "Initial Design: Sobol",
    "9": "Initial Design: Sobol",
    "10": "Initial Design: Sobol",
    "11": "Initial Design: Sobol",
    "12": "Initial Design: Sobol",
    "13": "Initial Design: Sobol",
    "14": "Initial Design: Sobol",
    "15": "Initial Design: Sobol",
    "16": "Initial Design: Sobol",
    "17": "Initial Design: Sobol",
    "18": "Initial Design: Sobol",
    "19": "Initial Design: Sobol",
    "20": "Initial Design: Sobol",
    "21": "Initial Design: Sobol",
    "22": "Initial Design: Sobol",
    "23": "Initial Design: Sobol",
    "24": "Acquisition Function Maximizer: Local Search",
    "25": "Acquisition Function Maximizer: Local Search",
    "26": "Acquisition Function Maximizer: Local Search",
    "27": "Acquisition Function Maximizer: Local Search",
    "28": "Acquisition Function Maximizer: Local Search",
    "29": "Acquisition Function Maximizer: Local Search",
    "30": "Acquisition Function Maximizer: Local Search",
    "31": "Acquisition Function Maximizer: Local Search",
    "32": "Acquisition Function Maximizer: Local Search",
    "33": "Acquisition Function Maximizer: Local Search",
    "34": "Acquisition Function Maximizer: Local Search",
    "35": "Acquisition Function Maximizer: Local Search"
  }
}